src/server/handlers/store.js

```diff
@@ -0,0 +1,22 @@
+import 'isomorphic-fetch';
+
+import { conf } from '../helpers/config';
+
+export const registerName = (req, res) => {
+  const snapName = req.body.snap_name;
+  const macaroon = req.body.macaroon;
+
+  return fetch(`${conf.get('STORE_API_URL')}/register-name/`, {
+    method: 'POST',
+    headers: {
+      'Authorization': `Macaroon root="${macaroon}"`,
+      'Content-Type': 'application/json',
+      'Accept': 'application/json'
+    },
+    body: JSON.stringify({ snap_name: snapName })
+  }).then((response) => {
+    return response.json().then((json) => {
+      return res.status(response.status).send(json);
```

Comment:
For sake of consistency it may be worth using same response pattern we use elsewhere: `{ status: 'success', payload: 'response data'}`.

I also assume we don't treat errors in any special way, we just pass them to client as they are?